spec/binaryTree_test.js

```diff
@@ -0,0 +1,66 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinaryTree from '../src/binaryTree'
+
+chai.use(chaiChange)
+
+describe('binaryTree', () => {
+  it('exists', () => {
+    expect(BinaryTree).to.be.a('function')
+  })
+
+  context('insert()', () => {
```

Comment:
Did you have to graph out this search tree before writing the tests? I am super curious.